### ISSUE TITLE
Can I use your scripts in my server?

### ISSUE THREAD

**Issue by mercury233** [Ygoproco/Live#32]
I'm a Chinese developer. I host a server of ygopro, this is the repository: 
https://github.com/mercury233/ygopro
https://github.com/Daniel17173/ygopro

What licenses do you use for your scripts? Can I use your scripts in my server?

**Comment by Percival18** [comment]
Yes you can use them as long as the script writers are credited.

**Comment by Zayelion** [comment]
Same cool for Salvation?

**Comment by Percival18** [comment]
Yep of course.

**Comment by Zayelion** [comment]
Blue-Eyes Spirit Dragon and Bloom Prima **[https://github.com/SalvationDevelopment/YGOPro-Salvation-Server/commit/5c7913edd765b910499c7a99652a5a6f892cd48c ]**

The scripts are here : https://github.com/SalvationDevelopment/Salvation-Scripts-TCG 
Like this repo, updates `[master]` to it are live to production.

**Comment by mercury233** [comment]
Thank you!

**Comment by Ygoproco** [comment]
Please also mention the website http://www.ygopro.co thanks.

**Comment by Zayelion** [comment]
Are we sharing bug fixes?

**Comment by mercury233** [comment]
Credit added in [github repository](https://github.com/Daniel17173/ygopro) and [release page](http://mercury233.me/ygosrv233/pre.html) and patch file(in next update).
